cartridges/demo_app_training/cartridge/templates/default/search/components/productTiles.isml

```diff
@@ -0,0 +1,7 @@
+<isloop items="${pdict.productSearch.productIds}" var="product">
+    <div class="col-6 col-sm-4">
+        <isobject object="${product.productSearchHit}" view="searchhit">
+            <isinclude url="${URLUtils.url('Tile-Show', 'pid', product.productID, 'swatches', true, 'ratings', true)}" />
+        </isobject>
+    </div>
+</isloop>
```

Comment:
this is not related to this practical exercise

Reply:
removed in d7d12885e6cc5dd941a95d176704274bb594f3c8